augur/filter/__init__.py

```diff
@@ -68,6 +69,12 @@ def register_arguments(parser):
     output_group.add_argument('--output-metadata', help="metadata for strains that passed filters")
     output_group.add_argument('--output-strains', help="list of strains that passed filters (no header)")
     output_group.add_argument('--output-log', help="tab-delimited file with one row for each filtered strain and the reason it was filtered. Keyword arguments used for a given filter are reported in JSON format in a `kwargs` column.")
+    output_group.add_argument(
+        '--empty-results-reporting',
+        type=EmptyResultsReportingMethod,
+        choices=[ method for method in EmptyResultsReportingMethod ],
```

Comment:
_non-blocking_

Does

```suggestion
        choices=list(EmptyResultsReportingMethod),
```

work?

Reply:
Ah, yup it does work. The list comprehension was left over from when the class was just a basic `Enum` instead of the custom `ArgparseEnum`.

Reply:
Updated in force push.